bin/runtests

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env php
+<?php
+require('runtests.php');
```

Comment:
Because one is a php file and one is an executable file

On Fri, Mar 25, 2016 at 05:15:45AM -0700, Daniel Rotter wrote:
>    In [1]bin/runtests:
> 
>  > @@ -0,0 +1,3 @@
>  > +#!/usr/bin/env php
>  > +<?php
>  > +require('runtests.php');
> 
>    Why don't we simply run runtests.php?
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu/pull/2184#discussion_r57439561
>    2. https://github.com/sulu-io/sulu/pull/2184/files/fe0af7bd20521c6e7cc60d5f84d163d3d6cf1997#r57439561
>    3. https://github.com/sulu-io/sulu/pull/2184/files/fe0af7bd20521c6e7cc60d5f84d163d3d6cf1997#r57439561